Compilers.slnf

```diff
@@ -2,18 +2,6 @@
   "solution": {
     "path": "Roslyn.sln",
     "projects": [
-      "src\\Analyzers\\CSharp\\Analyzers\\CSharpAnalyzers.shproj",
```

Comment:
Do we also need to remove these from Compilers.sln?

Reply:
@mavasani Probably it's okay to remove them.

They exist in Compilers.sln but don't even show up in solution explorer (if I remember correctly) for some reason.